src/Api.js

```diff
@@ -4,10 +4,16 @@ const newsApi = axios.create({
   baseURL: "https://earlynews.herokuapp.com/api",
 });
 
-export const fetchArticles = () => {
-  return newsApi.get("/articles").then((res) => {
-    return res.data.articles;
-  });
+export const fetchArticles = (sortBy, order) => {
+  console.log("inside api call", `articles?sort_by=${sortBy}&order=${order}`);
+  return newsApi
+    .get(`/articles?sort_by=${sortBy}&order=${order}`)
+    .then((res) => {
+      return res.data.articles;
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
do you mean to do something with this err?

Reply:
Was trying to debug an error, but I will handle it on task 12.